### PR TITLE
Add option to automically add webp image formats

### PR DESCRIPTION
**Pull request by alexander-schranz**
Use the following way to output all images also as webp:

```yaml
services:
    Sulu\Twig\Extensions\ImageExtension:
        arguments:
            $additionalTypes:
                webp: 'image/webp'
```

### CODE REVIEW

**Comment by alexander-schranz** [comment]
To-Do: ignore webp when svg ist rendered

File: docs/image.md [hunk 1]
```diff
@@ -183,3 +183,47 @@ services:
...
+
+If your server supports converting image to webp you can automatically enable the webp output for all images
+the following way:
+
+```yaml
+services:
+    Sulu\Twig\Extensions\ImageExtension:
+        arguments:
+            $additionalTypes:
+                webp: 'image/webp'
```
Comment (alexander-schranz) [inline docs/image.md:197]: I did go with this format as maybe in future we could add support for [heif](https://de.wikipedia.org/wiki/High_Efficiency_Image_File_Format) or other new formats this way.

File: docs/image.md [hunk 2]
```diff
@@ -183,3 +183,47 @@ services:
             $defaultAttributes:
                 loading: 'lazy'
 ```
+
+##### 7. Webp Support
+
+If your server supports converting image to webp you can automatically enable the webp output for all images
```
Comment (nnatter) [inline docs/image.md:189]: `enable webp output for the get_image twig extension per default`

File: src/ImageExtension.php [hunk 3]
```diff
@@ -42,16 +48,38 @@ class ImageExtension extends AbstractExtension
...
     /**
      * @param string[] $defaultAttributes
+     * @param string[] $additionalTypes
      */
-    public function __construct(?string $placeholderPath = null, array $defaultAttributes = [])
-    {
+    public function __construct(
+        ?string $placeholderPath = null,
+        array $defaultAttributes = [],
+        array $additionalTypes = []
```
Comment (nnatter) [inline src/ImageExtension.php:75]: maybe `$defaultAdditionalTypes` (we also use `defaultAttributes `)- but not sure if you did this intentionally

Comment (alexander-schranz) [reply src/ImageExtension.php:75]: I did it intentionally but I'm okay to keep it the same as above.

File: docs/image.md (hunk 2 above)
Comment (alexander-schranz) [reply docs/image.md]: @nnatter Updated the text a little bit as it doesn't only affect the get_image function.

**Comment by nnatter** [comment]
dont have the permissions to merge this, but looks good to me 🙂

**Pull request merged**